include/onnxruntime/core/framework/run_options.h

```diff
@@ -7,6 +7,7 @@
 #include <atomic>
 #include "core/session/onnxruntime_c_api.h"
 #include "core/framework/config_options.h"
+#include "core/session/onnxruntime_run_options_config_keys.h"
```

Comment:
This file is not really needed here.

Reply:
I moved the include to utils.cc where it actually is needed. I thought it might make sense in run_options.h.